patient-appointment-management/assets/aws/send_appointment_reminders.private.js

```diff
@@ -0,0 +1,345 @@
+/* eslint-disable camelcase */
```

Comment:
These should, I believe, work as one line: `/* eslint-disable camelcase complexity sonarjs/cognitive-complexity */`

Reply:
Oh, I tried that can I got an error, so I thought it didn't work. Just tried again and it fails when one line... not sure if I'm doing something wrong.

Reply:
Ah, my example was missing commas. See https://learn.coderslang.com/0023-eslint-disable-for-specific-lines-files-and-folders/#disabling-specific-eslint-rules

Reply:
gotcha. will change in the next commit/push round.